# BrushedMotorCommandMsg
uint8 part_id
uint8 id
uint8 current_limit
uint8 pwm

# See definitions in BrushedMotor.h
